python/github_repo_cloner/clone_handler.py

```diff
@@ -0,0 +1,121 @@
+"""Module for CloneHandler class"""
+import subprocess
+import math
+import requests
+
+class CloneHandler:
+    """Github Clone Handler"""
+    PER_PAGE = 30.0
+    BASE_URL = 'https://api.github.com'
+
+    def __init__(self, username):
+        self.username = username
+        self.user_info = {}
+        self.repo_info = []
+        self.errors = []
+
```

Comment:
@murjax , how do you order your methods in a class?

1. Is there a logic to it like the items at the  bottom are executed before the ones at the top?
2. Or do you order alphabetically when there is no reason to have any order?

3. Or just write the function where ever when you have a need for it?

Reply:
<img width="1168" alt="Screenshot 2023-09-29 at 3 34 49 PM" src="https://github.com/murjax/Github-Repo-Cloner/assets/11463275/8f43544a-43b6-4d0f-b3c2-88e96457baba">

Reply:
Mostly 3. I try to group methods close to where they are used. I don't follow it consistently though. Sometimes there are similar looking methods that are better grouped together but are used in different places.